boards/mips-malta/malta.c

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2016, Imagination Technologies Limited and/or its
+ *                 affiliated group companies.
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+
+
+#include <stdio.h>
+
+#include "periph/uart.h"
+
+#define MIPS_MALTA_ADDR 0xbf000500
+#define MIPS_MALTA_VAL_RST 0x42
```

Comment:
Sorry for the confusing comments. We had a discussion about the use of parens here some time ago and the conclusion was that we leave it up to the developer if he/she prefers parens or not. IMO the most important thing is consistent use of parens and I had the impression that the use in this PR seemed pretty arbitrary. However, I have to admit that I haven't checked thoroughly if the use is consistent within the support files for a certain platform.

Long story short: this is mostly a matter of readability and personal taste and not a blocker at all for this PR.